pkg/api/types.go

```diff
@@ -46,7 +46,7 @@ type APIVersions struct {
 // Scanner interface that all scanners should define.
 type Scanner interface {
 	// Scan will scan the image
-	Scan(string, *docker.Image) error
+	Scan(path string, image *docker.Image) error
```

Comment:
@enoodle we discussed that, we should first "standardize" what the resource should look like to cover all scan results...